docker-compose-3-persona.yml

```diff
@@ -135,6 +135,9 @@ services:
       - IPFS_HOST=ipfs
       - IPFS_PORT=5001
       - USER_URI=//Alice
+      - API_SWAGGER_BG_COLOR=#e4f2f3
+      - API_SWAGGER_TITLE=API-Heidi
```

Comment:
Do we need 'API-xxx' here, or just the names?

Reply:
Just the name but keep in mind , the dev doing the demo will have another tab open for identity and we need to tell them apart ..